documentation/contributing/contributor-guide.adoc

```diff
@@ -54,7 +54,16 @@ image:operators.png[Operators within the Strimzi architecture]
 [[contributing]]
 == Contributing to the documentation
 
-Once you have your local repository set up and have up to date copies of upstream content you need to follow these steps to contribute to the documentation.
+Once you have your local repository set up and have up-to-date copies of upstream content,
+follow these steps to contribute to the documentation.
+
+You add content to the documentation hosted on GitHub using pull requests (PRs).
+
+The PR must be reviewed before it can be merged.
+Request a Subject Matter Expert (SME) review from someone who is familiar with the technical aspects of the content.
+You might not require an SME review if you're only fixing a typo or broken link.
+
+Request additional peer reviewers for general checks on the content.
```

Comment:
Technically, this is right. But it is not really useful because new contributors for whose this might be relevant do not know who the SME is and do not really have any Strimzi peers to do the review.

Assuming we do not want to maintain a list of SMEs, maybe this should be written in passive voice. That the review from SMEs will be requested. At the end, regular users can anyway not request reviews on PRs IIRC. I would probably leave out the peers.

Reply:
Maybe we can add a note to say something like: _If you don't know who to assign for review, add a comment for  @\strimzi/maintainers to request assistance_

Reply:
I would expect every maintainer to be subscribed to the notifications about new PRs.

Reply:
Changed my mind. As we're dropping all the label stuff, this is probably sufficient:

> Reviewers might be assigned to the PR depending on the changes.
> A review from a Subject Matter Expert (SME) will check the technical aspects of the content.
> The PR might not require an SME review if you're only fixing a typo or broken link.